extensions/git/package.json

```diff
@@ -265,6 +265,11 @@
         "title": "%command.fetch%",
         "category": "Git"
       },
+      {
+        "command": "git.fetchPrune",
+        "title": "%command.fetchPrune%",
+        "category": "Git"
+      },
       {
         "command": "git.fetchAll",
         "title": "%command.fetchAll%",
```

Comment:
also don't forget to add condition in `menus.menus.commandPalette`

Reply:
Thanks @oneslash, I definitely missed this